### ISSUE TITLE
fix security vulnerabilities by upgrading dependencies

### ISSUE THREAD

**Issue by smirceal** [Vantiv/cnp-sdk-for-java#26]
Tried pulling latest from maven central and we see security warning for several dependencies.

**Could you please check if updating this dependencies is an option?**
 * **org.bouncycastle:bcprov-jdk15on** need to updated to **1.66** or higher 
   * https://app.snyk.io/vuln/SNYK-JAVA-ORGBOUNCYCASTLE-32369
   * https://app.snyk.io/vuln/SNYK-JAVA-ORGBOUNCYCASTLE-32412
   * https://app.snyk.io/vuln/SNYK-JAVA-ORGBOUNCYCASTLE-1035561
   * https://app.snyk.io/vuln/SNYK-JAVA-ORGBOUNCYCASTLE-1296075
 * **org.bouncycastle:bcprov-ext-jdk15on** needs to be updated to **1.66** or higher 
   * https://app.snyk.io/vuln/SNYK-JAVA-ORGBOUNCYCASTLE-474616
   * https://app.snyk.io/vuln/SNYK-JAVA-ORGBOUNCYCASTLE-1035564
   * https://app.snyk.io/vuln/SNYK-JAVA-ORGBOUNCYCASTLE-1296076
 * **commons-io:commons-io** needs to be updated to **2.7** or higher
   * https://app.snyk.io/vuln/SNYK-JAVA-COMMONSIO-1277109
 *  **junit:junit** needs to be updated to **4.13.1** or higher
    * https://app.snyk.io/vuln/SNYK-JAVA-JUNIT-1017047

**Comment by cwood19** [comment]
Also having your deployed artifact depending directly on junit and mockito is suboptimal:

From https://repo1.maven.org/maven2/io/github/vantiv/cnp-sdk-for-java/12.19.1/cnp-sdk-for-java-12.19.1.pom:
```
<dependency>
  <groupId>junit</groupId>
    <artifactId>junit</artifactId>
    <version>4.12</version>
    <scope>compile</scope>
</dependency>
<dependency>
  <groupId>org.mockito</groupId>
  <artifactId>mockito-all</artifactId>
  <version>1.9.5</version>
  <scope>compile</scope>
</dependency>
```

**Comment by ab9967** [comment]
Hi Folks, we have started looking into above issues & have fixed all security vulnerabilities in latest pull request [#27](https://github.com/Vantiv/cnp-sdk-for-java/pull/27).

Fix is in progress for junit & mockito dependencies. 

We are planning for a minor version release which you can expect in 2 weeks.

**Comment by ksd-15** [comment]
@ab9967 any update on this ?

**Comment by rohkulkagit** [comment]
@ksd-15 We are working on it and planning to resolve within this week.

**Comment by ksd-15** [comment]
@ab9967 @rohkulkagit we would appreciate ETA for the release of the security vulnerability fixes. We are blocked on this.

**Comment by rohkulkagit** [comment]
@ksd-15 
Due to other higher priority task, we can do this release  during the 1st week of December 2021

**Comment by ab9967** [comment]
We have released new version [12.19.2](https://search.maven.org/artifact/io.github.vantiv/cnp-sdk-for-java/12.19.2/jar)
Please check
